src/components/ScriptureWorkspaceCard.js

```diff
@@ -52,13 +53,21 @@ export default function ScriptureWorkspaceCard({
       console.log("useEffect() updated PERF:\n",JSON.stringify(_doc,null,4))
       const _usfm = await ep[docSetId].readUsfm( data.bookId.toUpperCase() )
 
-      await saveToUserBranch(
+      const _content = await saveToUserBranch(
         data,
         owner,
         _usfm,
         authentication,
         repoClient
       )
+      let _books = books
+      for ( let i=0; i<_books.length; i++ ) {
+        if ( _books[i].id = id ) {
```

Comment:
```suggestion
        if ( _books[i].id === id ) {
```
Really think you wanted a condition and not an assignment here.